core/tests/webdriverio_utils/workflow.js

```diff
@@ -257,7 +259,6 @@ var createSkillAndAssignTopic = async function(
   await topicsAndSkillsDashboardPage.get();
   await topicsAndSkillsDashboardPage.navigateToSkillsTab();
   await topicsAndSkillsDashboardPage.filterSkillsByStatus('Unassigned');
-  await topicsAndSkillsDashboardPage.searchSkillByName(skillDescription);
```

Comment:
Why was this removed?

Reply:
This was unnecessary as we already have searchSkillByName called at top in assignSkillToTopic funciton which we are calling just after this line.